ide/hudson.ui/src/org/netbeans/modules/hudson/ui/wizard/InstanceDialog.java

```diff
@@ -67,15 +67,15 @@ public void actionPerformed(ActionEvent e) {
     public HudsonInstance show() {
         dialog = DialogDisplayer.getDefault().createDialog(this);
         dialog.setVisible(true);
-        LOG.log(Level.FINE, "Added Hudson instance: {0}", created);
+        LOG.log(Level.FINE, "Added Jenkins instance: {0}", created);
         return created;
     }
 
     @NbBundle.Messages({
         "# UI logging of adding new server",
-        "UI_HUDSON_SERVER_REGISTERED=Hudson server registered",
+        "UI_JENKINS_SERVER_REGISTERED=Jenkins server registered",
```

Comment:
we going to have to check if we should change the keys and the values, or just the values

I think changing both should be fine, but lets see what others say.